sdk/test/Services/RestJsonProtocol/Generated/_netstandard/AmazonRestJsonProtocolClient.cs

```diff
@@ -2095,6 +2095,40 @@ internal virtual SimpleScalarPropertiesResponse SimpleScalarProperties(SimpleSca
 
         #endregion
         
+        #region  SparseJsonMaps
+
+        internal virtual SparseJsonMapsResponse SparseJsonMaps(SparseJsonMapsRequest request)
```

Comment:
These are only being generated because of the .json files that shouldn't be modified. Remove the JSON files, run the generator again and these will be removed. That will cause you to have to keep the partial files that were deleted.